deploy/helm/charts/templates/csidriver.yaml

```
@@ -8,3 +8,4 @@ spec:
   attachRequired: false
   podInfoOnMount: false
   storageCapacity: {{ .Values.feature.storageCapacity }}
+  seLinuxMount: true
```

Maybe set this via helm variable?

---

moreover if this is supported in Kubernetes version ≥ 1.27  then why are we setting it to true as default

---

It seems to be accepted from 1.25.x, so I suggest this:

```suggestion
{{- if semverCompare ">= 1.25.x" .Capabilities.KubeVersion.Version }}
  seLinuxMount: true
{{- end }}
```

---

thank you